src/makeCalculator.js

```diff
@@ -37,7 +37,39 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+
+    reset() {
+      this.result = 0;
+
+      return this;
+    },
+
+    operate(callback, operand) {
+      this.result = callback(this.result, operand);
+
+      return this;
+    },
+
+    add: function(a, b) {
```

Comment:
<img width="1076" alt="image" src="https://github.com/mate-academy/js_advanced_calculator/assets/86789161/c434920f-254e-4f5f-a06e-a301365a25c0">

Reply:
And in all other places